.github/workflows/upload_package.yml

```diff
@@ -0,0 +1,68 @@
+
+name: Upload Python Package
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine jupyter
+      run: |
+        python setup.py sdist bdist_wheel
+    - name: Publish distribution 📦 to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        password: ${{ secrets.pypi_password }}
```

Comment:
just curious for future reference: you are storing your PyPI password as a GH secret in the `geosnap` repo?

Reply:
its an api key you create on pypi, then enter into the repository's settings. I followed https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/